File: python/字典(对象)/遍历字典/values.py
# 字典 values() 方法返回一个迭代器，可以使用 list() 来转换为列表，列表为字典中的所有值。

# dict.values()

# 返回值  返回迭代器。

Object = {
    "name": "Tom",
    "age": 18,
    "other": "其他"
}

print(Object.values()) # dict_values(['Tom', 18, '其他'])

for value in Object.values():
    print('value', value) # value Tom\value 18\value 其他

print(list(Object.values())) # ['Tom', 18, '其他']

File: javascript/对象/遍历对象/values.js
// values()

// 语法

// Object.values()

// 详情

// 方法返回一个数组，成员是参数对象自身的（不含继承的）所有可遍历（enumerable）属性的键值。

// 示例

let object = {
    name: "Tom",
    age: 18
}


let object_values = Object.values(object);

console.log(object_values); // ["Tom", 18]

